lang/csharp/src/apache/test/Schema/ArraySchemaTests.cs

```diff
@@ -0,0 +1,37 @@
+using NUnit.Framework;
+
+namespace Avro.test
+{
+    [TestFixture]
+    public class ArraySchemaTests
+    {
+        [Test]
+        public void EqualsNullCheck()
+        {
+            string schemaString = "{\"type\": \"array\", \"items\": \"long\"}";
+            ArraySchema nullSchema = null;
+
+            Schema schema = Schema.Parse(schemaString);
+
+            if (schema is ArraySchema arraySchema)
+            {
+                Assert.False(arraySchema.Equals(nullSchema));
+            }
+            else
+            {
+                Assert.Fail("Schema was not an Array Schema");
```

Comment:
`Array Schema must not be equal to null` ?!

Reply:
Fixed